web/gui/goto-host-from-alarm.html

```diff
@@ -101,6 +105,8 @@
         + ';alarm_unique_id=' + urlOptions.alarm_unique_id.toString()
         + ';alarm_id=' + urlOptions.alarm_id.toString()
         + ';alarm_event_id=' + urlOptions.alarm_event_id.toString()
+        + ';after=' + alarm_time.toString()
+        + ';alarm_freeze=' + urlOptions.alarm_freeze.toString()
```

Comment:
so, to summarize slack discussion, i think we don't need to have special case for `alarm_freeze`. Dashboard is handling `before` and `after` properties and this should be enough - the freeze problem was probably caused because you omitted `before` property. To have more clean code, i think it's best to calculate the -/+ 2 minutes directly in goto-host-from-alarm.html

Reply:
I could not work with the before correctly, I remember that every time I tried to change it I did not have success, this was the one of the motives I created the alarm_freeze.

Thanks by your tips.